pex/bin/pex.py

```diff
@@ -496,6 +496,16 @@ def configure_clp():
       help='Add constraints from the given constraints file.  This option can be used multiple '
            'times.')
 
+  parser.add_option(
+    '--requirements-pex',
+    dest='requirements_pexes',
+    metavar='FILE',
+    default=[],
+    type=str,
+    action='append',
+    help='Add requirements from the given .pex file or unzipped pex directory.  This option can '
```

Comment:
Even if this happens to work on an unzipped PEX, there is no way to create one from the CLI so this seems like a confusing bit to document in the context of a CLI option as opposed to the PEXBuilder API where it does make sense.

Reply:
Well, you can unzip a `.pex` file to get an unzipped pex directory... But I guess Pants now uses zipped `.pex` files exclusively, so maybe there's no need to document this.